Documentation/diagnostics/source_link.md

```diff
@@ -0,0 +1,103 @@
+# Source Link
+[Portable PDB](https://github.com/dotnet/core/blob/master/Documentation/diagnostics/portable_pdb.md) introduces a new developer productivity feature called Source Link. Source Link allows a Portable PDB to embed unique information about the assembly's original source code during compilation. The [Roslyn](https://github.com/dotnet/roslyn/wiki/Roslyn%20Overview) compiler flag for Source Link is `/sourcelink:<file>`.
```

Comment:
It's coming, but we are not sure yet when we will be able to ship the Windows PDB support.